updates/julian_1.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE einzelbeleg_hattitelkritik ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE einzelbeleg_hattitelkritik ADD provenance_id VARCHAR (255) DEFAULT NULL, ADD CONSTRAINT unique_provenance_id UNIQUE (provenance_id);
+
+ALTER TABLE person_hatgruppeherkunftareal ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE person_hatgruppeherkunftareal ADD provenance_id VARCHAR (255) DEFAULT NULL, ADD CONSTRAINT unique_provenance_id UNIQUE (provenance_id);
+
+ALTER TABLE selektion_arealtyp ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
```

Comment:
These columns already existed on my local machine. Are they already part of any other SQL patch?

Reply:
yes selektion_arealtyp has already the provenance columns. I have removed the statement